orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/DependentPipelineStarter.groovy

```diff
@@ -88,6 +91,14 @@ class DependentPipelineStarter implements ApplicationContextAware {
     def t1 = new Thread(new Runnable() {
       @Override
       public void run() {
+        if (user) {
+
+          def securityCtx = SecurityContextHolder.createEmptyContext()
+          // Using RunAsUserToken is an arbitrary Authentication objection - it's merely named the best for this use case.
```

Comment:
The problem is that `DependentPipelineExecutionListener` is running on a separate thread from independent pipeline, so there is nothing in the SecurityContext/MDC mechanism for `Authenticatedreuest.propagate()` to read values from.

To put it a bit more clearly, we're dealing with 3 separate threads:

1. The original/independent pipeline thread
2. The `DependendPipelineExecutionListener` thread
3. The Thread created within 2.) to run the dependent pipeline.

The security context needs to propagate from 1 -> 3, but is being dropped between 1 and 2, and I don't control the invocation, so I can't wrap it with `propagate`